src/FixerDescriptionAwareInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+interface FixerDescriptionAwareInterface
+{
+    /**
+     * @return FixerDescription
+     */
+    public function getFixerDescription();
```

Comment:
that's the thing we are doing now for extended description ;)
i just afraid to have fixers without _any_ descriptions... very well, let it be. in that way we should focus to make that change really fast, to introduce new BC breaker as fast as possible